README.md

```diff
@@ -141,41 +141,37 @@ separately.
 - [Releases Page](RELEASES.md): Documentation for how to leverage our build artifacts.
```

Comment:
Actually, maybe you can merge this new documentation into https://github.com/ROCm/TheRock/blob/main/RELEASES.md ? How different is the process there from what this script does? There is overlap here.